### ISSUE TITLE
too many * (stop codons)

### ISSUE THREAD

**Issue by flystar233** [agshumate/Liftoff#60]
There are too many * (stop codons) in the protein sequence，after the gff file is converted into a protein file.
the target : Tanichthys albonubes （ Cyprinidae）
the refer：Danio rerio （ Cyprinidae）
![](https://i.loli.net/2021/01/15/GKzp1NkFDXA85WV.jpg)

**Comment by agshumate** [comment]
likely the target assembly has an insertion or deletion within the coding part of the transcript leading to this frameshift. to get a better idea, check out the .sam alignments in the intermediate_files directory. Also it is worth confirming that the original reference transcript does not have internal stop codons as well. While this should not be the case, it does happen occasionally.

**Comment by yuzhenpeng** [comment]
> did you use the -polish option?

Yes, I used it.

**Comment by agshumate** [comment]
can you elaborate on what you mean when you checked the genome sequencing reads? do you mean the reads aligned to your new genome?

**Comment by yuzhenpeng** [comment]
Hello,
Sorry for my poor english. 

I used the Liftoff to annotate my new genome using its closest relatives genome with high-quality. But, I found that there are a lot stop codon. 

Next, I used the blast method to search the same coding sequence in my new genome using the sequence of closest relatives as query. Howerver, it is not have stop codon.

Final, I  checked the original sequencing data 150 PE reads (not assembly) of my genome. The results is same as blast results，no stop codon in genome.
